src/test/groovy/generated/AbstractGeneratedAstTestCase.groovy

```diff
@@ -0,0 +1,60 @@
+package groovy.generated
+
+import groovy.transform.CompileStatic
+import groovy.transform.Generated
+
+import java.lang.reflect.Constructor
+import java.lang.reflect.Method
+
+/**
+ * @author Dmitry Vyazelenko
+ * @author Andres Almiray
```

Comment:
Apache generally discourages explicit @author tags. They get out of date quickly, provide less info than git will already tell you and can sometimes discourage participation if certain files are perceived to belong to particular individuals rather than the whole community. I'd suggest removing them but add Dmitry as contributor in pomconfigurer,gradle - you are already there. :-)